Directory.Build.props

```diff
@@ -7,8 +7,25 @@
 
     <!-- This is used to disable some build properties. -->
     <IsExamplesSolution Condition="'$(SolutionName)' == 'Examples' ">true</IsExamplesSolution>
+
+    <!-- This is used to change EventSource names. -->
+    <DefineConstants Condition="'$(Redfield)' == 'True'">$(DefineConstants);REDFIELD</DefineConstants>
+  </PropertyGroup>
+
+  <PropertyGroup Label="Package versions for System.Diagnostics.DiagnosticSource">
+    <SystemDiagnosticsDiagnosticSourcePkgVer>5.0.0</SystemDiagnosticsDiagnosticSourcePkgVer>
```

Comment:
5.0.1 would be ideal as it has critical timestamp accuracy fix.

Reply:
This is out of the scope for this PR as we are not changing any packages related to the actual flow. Do you want to change the version in PR?